common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -194,58 +194,76 @@ describe 'Problem', ->
 
     describe 'when the response is correct', ->
       it 'call render with returned content', ->
-        spyOn($, 'postWithPrefix').andCallFake (url, answers, callback) ->
-          callback(success: 'correct', contents: 'Correct!')
```

Comment:
`Correct!` is not a valid selector. Please check http://stackoverflow.com/questions/13979323/how-to-test-if-selector-is-valid-in-jquery